BrownField/Networking/Concepts/Connectivity-Multi-region-with-vWAN/Dual-Region-with-vWAN.md

```diff
@@ -0,0 +1,122 @@
+**Brown Connections**: AVS private cloud connection to its local regional hub.  
+**Pink Connections**: AVS private cloud connection to its cross-regional hub.   
+**Orange Connection**: AVS Region 1 Global Reach connection back to on-premise.   
+**Green Connection**: AVS Region 2 Global Reach connection back to on-premise.   
+**Purple Connection**: AVS Region 1 and AVS Region 2 Global Reach connection back to each other's private cloud.   
+**Black Connections**: On-premise connectivity via ExpressRoute to both regional hubs.  
+**Inter-Hub Connection**: When two hubs are deployed under the same vWAN, they automatically form an inter-hub connection with one another. The purpose of the inter-hub is to transit cross-regional traffic between the two hubs.  
+
+## Dual-region Secure vWAN Traffic Flows
+
+The following sections below will discuss traffic flows and connectivity for AVS, On-Premise, Azure Vnets, and the Internet. 
+
+### Azure VMware Solution cross-region connectivity & traffic flows
+
+This section will focus on only the AVS Cloud Region 1 and AVS Cloud Region 2. Each AVS private cloud will have an ExpressRoute connection to its local regional hub (brown connections) and an ExpressRoute connection to the cross-regional hub (pink connections).
+
+Each AVS Cloud Region connects back to on-premise via Global Reach. AVS Cloud Region 1 Global Reach connection is shown in orange as "Global Reach (A)". AVS Cloud Region 2 Global Reach connection is shown in green as "Global Reach (B)". Both AVS private clouds are connected directly to each other via Global Reach shown in purple as Global Reach (C). Keep in mind that Global Reach traffic will never transit any hub firewalls. See traffic flow section below for more information.  
+
+The diagram also depicts how all routes in each AVS Cloud region will learn routes from both the local and cross-regional hub. All blue routes are from Region 1, and all red routes are from Region 2. 
+
+![image](https://github.com/jasonamedina/Enterprise-Scale-for-AVS/assets/97964083/43d9ac83-d982-4cd1-8760-ff32b5dd6f76)
+
+**Traffic Flow**
+| From |   To |  Hub 1 VNets | On-Premise | Hub 2 VNets | Cross-Regional AVS Private Cloud|
+| -------------- | -------- | ---------- | ---| ---| ---|
+| AVS Cloud Region 1    | &#8594;| Hub1Fw>Vnet1|  GlobalReach(A)>OnPremise   | Hub2Fw>Vnet2 | Global Reach(C)>AVS Cloud Region 2|
+| AVS Cloud Region 2   | &#8594;|  Hub1Fw>Vnet1 |  GlobalReach(B)>OnPremise   | Hub2Fw>Vnet2 | Global Reach(C)>AVS Cloud Region 1|
+
+### On-Premise connectivity & traffic flow
+
+This section will focus only on the on-premise site. As shown in the diagram below, the On-Premise site will have an ExpressRoute connection to both Region 1 and Region 2 hubs (black connections).
+
+On-Premise can communicate to AVS Cloud Region 1 via orange connection "Global Reach (A)". On-Premise will also be able to communicate with AVS Cloud Region 2 via green connection "Global Reach (B).
```

Comment:
missing double quote at the end of the sentence

Reply:
@fguerri, I made the correction shown highlighted below.
![image](https://github.com/Azure/Enterprise-Scale-for-AVS/assets/97964083/894bed35-7c1a-4e39-a77c-c7dedbea7dec)